main.go

```diff
@@ -115,15 +115,23 @@ func main() {
 		log.Fatal(err)
 	}
 
+	policies := []plan.Policy{}
+
+	// if ownership enabled
+	if cfg.Registry == "txt" {
+		policies = append(policies, &plan.OwnershipPolicy{Owner: cfg.TXTOwnerID})
```

Comment:
there is the same `if` check above in a switch statement. Might be better to place it there